packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -100,11 +112,13 @@ export function newProject(options: OptionsSchema): Rule {
 						}
 					},
 					(tree: Tree, context: IgxSchematicContext) => {
-						return defer(async () => {
-							prompt.setContext(context, tree, options.name as string);
-							await prompt.chooseActionLoop(projLibrary);
-							return tree;
-						});
+						if (!allOptionsProvided || !options.defaults) {
```

Comment:
I think this check should just be `!allOptionsProvided`
If you've provided `theme`, `name` & `template`, it's still possible that `!option.defaults === true` and this still enters.
If `options.defaults === true`, then `allOptionsProvided === true`, so it won't enter